tests/yam/agama/import_agama_profile.pm

```diff
@@ -0,0 +1,20 @@
+## Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Run Agama profile import on Live Medium
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base Yam::Agama::patch_agama_base;
+use strict;
+use warnings;
+use Utils::Architectures 'is_s390x';
+use testapi qw(assert_script_run data_url get_required_var select_console);
+
+sub run {
+    select_console 'root-console' unless is_s390x;
+    my $profile = get_required_var('AGAMA_PROFILE');
```

Comment:
we already have AGAMA_AUTO, we don't need to invent another setting.

Reply:
this conflicts with [boot_agama.pm](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/yam/agama/boot_agama.pm) where it is used for [automatic installation mode](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/yam/agama/boot_agama.pm#L41)